env/work/srw_python/uti_io.py

```diff
@@ -105,3 +105,69 @@ def write_ascii_data_cols(_file_path, _cols, _str_sep, _str_head=None, _i_col_st
         
     f.write(strTot)
     f.close()
+
+#********************** Read data from an image file:
+def read_image(image_path, bottom_limit=None, cutoff_background=0.5, ignore_bottom_limit=False): #MR17112016
+    """Read data from an image file.
+
+    :param image_path: full path to the image.
+    :param bottom_limit: the bottom limit separating the image and the legend (black block).
+    :param cutoff_background: the ratio for cutoff the background noise.
+    :return: dict with the processed data.
+    """
+
+    msg = '{0} library is not installed. Use "pip install {0}" to install it.'
+    try:
+        import numpy as np
+    except:
+        raise ValueError(msg.format('numpy'))
+    try:
+        from PIL import Image
+    except:
+        raise ValueError(msg.format('pillow'))
+
+    # Read the image:
+    raw_image = Image.open(image_path)
+    image_format = raw_image.format
+    raw_image = raw_image.convert('L')
```

Comment:
All supplied images are converted to 8-bit images to avoid the problems with RGB images.